client/common/i18n.js

```diff
@@ -114,6 +114,9 @@ class I18n {
     }
 
     systemLanguage() {
+
+        return 'es';
```

Comment:
I feel like this has been left in but accident

Reply:
good catch